CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# MobileCenter-SDK-iOS Change Log
+# mobile-center-sdk-ios Change Log
```

Comment:
Erg I though this one was already fixed, not a big deal though. Will be done in another PR.